ui/admin/app/components/form/target/details/index.hbs

```diff
@@ -218,81 +219,78 @@
+            />
+          {{else}}
+            <WorkerDiagram::SingleFilter
+              @egressFilter={{this.egressWorkerFilterEnabled}}
+            />
+          {{/if}}
 
-      {{#if this.showEgressFilter}}
-        <F.Control>
-          <WorkerDiagram::SingleFilter
-            @egressFilter={{this.egressWorkerFilterEnabled}}
-          />
-
-        </F.Control>
-        <F.Control>
-          <Form::Target::WorkerFilter
-            @name='egress_worker_filter'
-            @type='text'
-            @disabled={{form.disabled}}
-            @value={{@model.egress_worker_filter}}
-            @toggleAction={{this.toggleEgressWorkerFilter}}
-            @checked={{this.egressWorkerFilterEnabled}}
-            @toggleEnabled={{this.egressWorkerFilterEnabled}}
-            @errors={{@model.errors.egress_worker_filter}}
-            @onChange={{set-from-event @model 'egress_worker_filter'}}
-            @isInvalid={{@model.errors.egress_worker_filter}}
-            as |F|
-          >
-            {{#if @model.errors.egress_worker_filter}}
-              <F.Error as |E|>
-                {{#each @model.errors.egress_worker_filter as |error|}}
-                  <E.Message>{{error.message}}</E.Message>
-                {{/each}}
-              </F.Error>
-            {{/if}}
-          </Form::Target::WorkerFilter>
-
-        </F.Control>
-      {{/if}}
-
-      {{#if @model.isNew}}
-        <F.Control>
-          <WorkerDiagram::SingleFilter
-            @egressFilter={{this.egressWorkerFilterEnabled}}
-          />
-        </F.Control>
-        <F.Control>
-          <Form::Target::WorkerFilter
-            @name='egress_worker_filter'
-            @type='text'
-            @disabled={{form.disabled}}
-            @value={{@model.egress_worker_filter}}
-            @toggleAction={{this.toggleEgressWorkerFilter}}
-            @checked={{this.egressWorkerFilterEnabled}}
-            @toggleEnabled={{this.egressWorkerFilterEnabled}}
-            @errors={{@model.errors.egress_worker_filter}}
-            @onChange={{set-from-event @model 'egress_worker_filter'}}
-            @isInvalid={{@model.errors.egress_worker_filter}}
-            as |F|
-          >
-            {{#if @model.errors.egress_worker_filter}}
-              <F.Error as |E|>
-                {{#each @model.errors.egress_worker_filter as |error|}}
-                  <E.Message>{{error.message}}</E.Message>
-                {{/each}}
-              </F.Error>
-            {{/if}}
-          </Form::Target::WorkerFilter>
-        </F.Control>
-      {{/if}}
-
+          {{#if this.showOSSTargetWorkerFilters}}
+            <Form::Target::WorkerFilter
+              @name='egress_worker_filter'
+              @disabled={{form.disabled}}
+              @value={{@model.egress_worker_filter}}
+              @toggleAction={{this.toggleFilter}}
+              @checked={{this.egressWorkerFilterEnabled}}
+              @toggleEnabled={{this.egressWorkerFilterEnabled}}
```

Comment:
What's the difference?

Reply:
There's no difference; I used it for readability, but `@toggleEnabled` works in both scenarios, updated!